src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockGoogleServicesDAO.scala

```diff
@@ -82,4 +82,18 @@ class MockGoogleServicesDAO extends GoogleServicesDAO {
   override def removeUserFromProxyGroup(user: RawlsUser): Future[Unit] = Future.successful(mockProxyGroups += (user -> false))
 
   override def isUserInProxyGroup(user: RawlsUser): Future[Boolean] = Future.successful(mockProxyGroups.getOrElse(user, false))
+
```

Comment:
instead of passing in groupName to the functions below pass RawlsGroup or RawlsGroupRef. Insead of memberEmail, RawlsUser. Maybe add/remove methods for sub groups unless you want the api to take Either[RawlsUser, RawlsGroup]. In the case of user, need to proxyify.